.github/workflows/link-validator.yml

```diff
@@ -0,0 +1,37 @@
+name: Link Validator
+
+on:
+  pull_request:
+  schedule:
+    - cron:  '0 6 * * 1'
```

Comment:
Not super important but would be nice if we could manually trigger as well (given that they work as I think they do): 
```yaml
schedule:
     - cron:  '0 6 * * 1'
     - workflow_dispatch:
```